inc/languages/english/admin/user_users.lang.php

```diff
@@ -257,6 +257,9 @@
 $l['suspended_perm'] = "<p><small>Suspended permanently.<br />Enter a new time below to change or untick this option to remove this suspension.</small></p>";
 $l['suspend_length'] = "<p><small>Remaining Suspension: <span style=\"color: {2};\">{1}</span>.<br />Enter a new time below to change or untick this option to remove this suspension.</small></p>";
 
+$l['suspend_pm'] = "Suspend private messaging";
+$l['suspend_pm_info'] = "Suspend {1} from sending new private messages.";
```

Comment:
This explicit mention of _sending_ only would need to be changed as per https://github.com/mybb/mybb/issues/5134#issuecomment-3566929455